components/SponsorsSection.js

```diff
@@ -33,6 +34,7 @@ export default function TicketsSection () {
           {sponsorsSliver.map(name => (
             <ItemSponsor
               key={name}
+              name={name}
```

Comment:
This change looks like it is caused by a duplicate code. You can extract this logic into a function, `renderSponsor`, which can then be used like this:

```js
sponsorsSilver.map(renderSponsor)
```